core/src/main/resources/hudson/model/Cause/UserCause/description_zh_TW.properties

```diff
@@ -20,4 +20,4 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-started_by_user=\u7531\u4f7f\u7528\u8005 <a href="{1}/user/{0}">{0}</a> \u555f\u52d5
```

Comment:
This one is (probably, it looks like it but I cannot read it) a nice example that splitting out the user mention from the localized string doesn't always work because grammar.

Wouldn't just adding the `class` into each localized string do it?

Reply:
Yeah, that works out too.